### ISSUE TITLE
Setup testing infrastructure for metrics

### ISSUE THREAD

**Issue by aashish24** [UV-CDAT/uvcmetrics#133]
@aashish24 and @doutriaux1 will work on this.

**Comment by doutriaux1** [comment]
yes. Essentially moving ctests to this repo, create a dedicated baseline repo (???). 
setup bots to run test suite using a dedicated uvcdat (to speed up test time)

**Comment by susburrows** [comment]
awesome! :)
